README.md

```diff
@@ -17,7 +17,8 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 ## AWS CDK v1 vs AWS CDK v2
 Two separate versions of Datadog CDK Constructs exist; `datadog-cdk-constructs` and `datadog-cdk-constructs-v2`. These are designed to work with `AWS CDK v1` and `AWS CDK v2` respectively.
 
-- `datadog-cdk-constructs-v2` requires Node 14+, while `datadog-cdk-constructs-v1` supports Node 12+.
+- `datadog-cdk-constructs-v2` requires Node 14+, while `datadog-cdk-constructs` supports Node 12+.
```

Comment:
```suggestion
- `datadog-cdk-constructs-v2` requires Node >= 14, while `datadog-cdk-constructs` supports Node >= 12.
```
We are trying to standardize the docs on using ">=" to describe version constraints going forward.